hack/lib/test_lib.sh

```diff
@@ -42,34 +42,6 @@ function trap_add() {
     done
 }
 
-# add_go_mod_replace adds a "replace" directive from $1 to $2 with an
```

Comment:
checked that it is not used. https://github.com/operator-framework/operator-sdk/search?q=add_go_mod_replace&unscoped_q=add_go_mod_replace